src/soundio/soundmanager.cpp

```diff
@@ -344,7 +340,7 @@ SoundDeviceError SoundManager::setupDevices() {
     // results in a stuttering noise (sometimes a loud buzz noise at low
     // latencies) when changing devices.
     //m_pClkRefDevice = NULL;
-    m_pErrorDevice = NULL;
+    m_pErrorDevice.clear();
```

Comment:
Qt 5 will add reset() for compatibility with std::shared_ptr. Just a remark, nothing to do here.